<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="upDownButtonsFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="up_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="down_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>v</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="floor_spin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>45</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Floor:</string>
     </property>
    </widget>
    <widget class="QFrame" name="fcp_header">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Floor Control Panel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>661</width>
      <height>261</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>659</width>
       <height>259</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="scrollAreaHeader">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>OUTPUT LOG</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="elevator_informationFrame">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="elevator_spin">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>29</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Elevator:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="level_indicator">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(239, 41, 41);
border-color: rgb(0, 0, 0);
alternate-background-color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="weight_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Weight (Max 15T):</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="elevator_weight_spin">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Car Control Panel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="elevator_floorButtonsFrame">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="elevator_otherButtonsFrame">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="help_button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>HELP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_door_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>OPEN DOOR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_door_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>CLOSE DOOR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Car Buttons</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="setupFrame">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>No. of Cars</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>No. of Floors</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="num_cars_spin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>45</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="num_floors_spin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>45</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Safe Floor</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="safe_floor_spin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>45</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="elevator_externalButtons">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="fire_button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>71</width>
       <height>151</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>FIRE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="power_out_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>71</width>
       <height>151</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>OUTAGE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blockage_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>151</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="text">
      <string>BLOCKAGE</string>
     </property>
    </widget>
    <widget class="QFrame" name="external_signals_header">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(46, 52, 54);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>External Signals</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="info_header">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(46, 52, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>500</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 52, 54);</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>By: Gabriel Martell</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Elevator Simulation 3004</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>start_button</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
